server/src/suggestionBuilder.ts

```diff
@@ -6,12 +6,16 @@
 
 'use strict';
 
-import { TextDocumentPositionParams, TextDocument, CompletionItem, CompletionItemKind } from 'vscode-languageserver';
+import { TextDocumentPositionParams, TextDocument, CompletionItem, CompletionItemKind, IConnection, createConnection,
+    IPCMessageReader, IPCMessageWriter
+} from 'vscode-languageserver';
 import {
     TreeBuilder, FileNode, SymbolType, AccessModifierNode,
     ClassNode, TraitNode, MethodNode
 } from "./hvy/treeBuilder";
 
+let connection: IConnection = createConnection(new IPCMessageReader(process), new IPCMessageWriter(process));
```

Comment:
I was using it for debugging some things, I think I removed where I was doing the debugging.